internal/pkg/runtime/engines/imgbuild/rpc/client/client.go

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) 2018, Sylabs Inc. All rights reserved.
```

Comment:
Same here:

```suggestion
// Copyright (c) 2018-2019, Sylabs Inc. All rights reserved.
```

Or

```suggestion
// Copyright (c) 2019, Sylabs Inc. All rights reserved.
```